tests/kubeaddons-enterprise/addons/generic-application-operator/0.x/generic-operator-install-0-x/00-install.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl apply -f ${KUBEADDONS_ENTERPRISE_ABS_PATH}/addons/generic-application-operator/0.x/generic-application-operator.yaml
```

Comment:
Why don't we just symlink the file here?

Reply:
we could actually do it but using env variable felt easier as these tests are called from `kubeaddons-enterprise` repo CI step also